runtime/wasm/src/module/test.rs

```diff
@@ -595,13 +455,3 @@ fn big_int_arithmetic() {
     let result: BigInt = module.heap.asc_get(result_ptr);
     assert_eq!(result, BigInt::from(1));
 }
-
-#[test]
-fn abort() {
-    let mut module = test_module(mock_data_source("wasm_test/abort.wasm"));
```

Comment:
How come we're no longer able to test handling errors thrown by the module?

Reply:
hmm I'll try something with this one.

Reply:
Thanks for pointing this out, came up with a test that is better than the original.

Reply:
Great!